mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.java

```diff
@@ -146,6 +147,29 @@ public void onItemLongClicked(final Widget widget) {
                     labels.add(getString(R.string.nfc_action_current_color));
                     commands.add(widget.state().asString());
                 }
+            } else if (widget.type() == Widget.Type.Setpoint
+                    || widget.type() == Widget.Type.Slider) {
+                if (widget.state() != null) {
+                    ParsedState.NumberState state = widget.state().asNumber();
+
+                    String currentState = state.toString(Locale.getDefault());
```

Comment:
Need to null check state here? (e.g. by a ternary: `NumberState state = widget.state() != null ? widget.state().asNumber() : null;`)

Also can omit the locale parameter.

Reply:
I check widget.state() for null in line 152 already.

Reply:
`widget.state().asNumber()` can be null even with `widget.state` being != null: in case the state can't be converted to a number.